ts/src/coinlist.ts

```diff
@@ -0,0 +1,2246 @@
+        const quoteVolume = this.safeString (ticker, 'volume_quote_24h');
+        const high = this.safeString (ticker, 'highest_price_24h');
+        const low = this.safeString (ticker, 'lowest_price_24h');
+        const close = this.safeString (ticker, 'last_price');
+        const changePcnt = this.safeString (ticker, 'price_change_percent_24h');
+        return this.safeTicker ({
+            'symbol': market['symbol'],
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'open': undefined,
+            'high': high,
+            'low': low,
+            'close': close,
+            'bid': bid,
+            'bidVolume': undefined,
+            'ask': ask,
+            'askVolume': undefined,
+            'vwap': undefined,
+            'previousClose': undefined,
+            'change': undefined,
+            'percentage': changePcnt,
+            'average': undefined,
+            'baseVolume': baseVolume,
+            'quoteVolume': quoteVolume,
+            'info': ticker,
+        }, market);
+    }
+
+    async fetchOrderBook (symbol: string, limit: Int = undefined, params = {}) {
+        /**
+         * @method
+         * @name coinlist#fetchOrderBook
+         * @description fetches information on open orders with bid (buy) and ask (sell) prices, volumes and other data
+         * @param {string} symbol unified symbol of the market to fetch the order book for
+         * @param {int} [limit] the maximum amount of order book entries to return (default 100, max 200)
+         * @param {object} [params] extra parameters specific to the coinlist api endpoint
+         * @returns {object} A dictionary of [order book structures]{@link https://github.com/ccxt/ccxt/wiki/Manual#order-book-structure} indexed by market symbols
+         */
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'symbol': market['id'],
+        };
+        const response = await this.publicGetV1SymbolsSymbolBook (this.extend (request, params));
+        //
+        //     {
+        //         "bids": [
+        //             [ "30900.00000000", "0.0001" ],
+        //             [ "30664.21000000", "0.0172" ],
+        //             [ "30664.20000000", "0.0906" ],
+        //         ],
+        //         "asks": [
+        //             [ "31349.99000000", "0.0003" ],
+        //             [ "31350.00000000", "0.0023" ],
+        //             [ "31359.33000000", "0.0583" ],
+        //         ],
+        //         "after_auction_code": "BTC-USDT-2023-10-23T18:40:51.000Z",
+        //         "call_time": "2023-10-23T18:40:51.068Z",
+        //         "logical_time": "2023-10-23T18:40:51.000Z"
+        //     }
+        //
+        const logical_time = this.parse8601 (this.safeString (response, 'logical_time'));
+        const call_time = this.parse8601 (this.safeString (response, 'call_time'));
```

Comment:
call_time is not a nonce, we should first use only logical_time and second set nonce as undefined